pynestml/codegeneration/resources_nest/directives/MemberInitialization.jinja2

```diff
@@ -9,10 +9,4 @@
 {%-   else %}
 {{printer.print_origin(variable)}}{{names.name(variable)}} = {{printer.print_expression(variable.get_declaring_expression())}}; // as {{variable.get_type_symbol().print_symbol()}}
 {%-   endif %}
-{%- else %}
```

Comment:
Should these lines of change be in this PR, or do they come from the vector PR?

Reply:
This is the part of the code that initialized the state variables to 0 by default before if they were not initialized. Now that we have a CoCo defined for it, I removed this code.

Reply:
I see. All good!